### DISCUSSION TITLE
Does Alpine.js have a formal browser compatibility list?

### DISCUSSION THREAD

**Discussion by lightwalkernet** [alpinejs/alpine#4452] [1. Help]
Having run into a few issue recently with Livewire and Alpine from clients using older browsers and noticing that alpine does not work at all on ios 13 (I know very few still use it) but it would be helping to have a clear browser compatibility list so that we can show clients to determine if alpine/livewire is the best tool for their project?

**Comment by ekwoka** [comment]
If you bundle it yourself, it can pretty much safely target ES2017.

I don't think any features are not safely targeted to that.

If you have the goal of targeting dead browsers, you really NEED to be bundling code yourself.
